### DISCUSSION TITLE
Not seeing Security Onion host in TheFleet + not seeing my Windows 10 PC in TheFleet either

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#1575] [Unsupported Versions]
So I'm using the provided ISO for Security Onion 2.3.0, fresh install, not coming from a release client. I believe during the conference it was mentioned you would see Security Onion's own host in TheFleet. I've also installed the provided .msi under the downloads menu on my Windows 10 PC. I installed it after running so-allow and choosing the osquery endpoint option and giving it my IP. I do not see either the Security Onion host or my Windows 10 PC in TheFleet.

I am seeing the install on my Windows 10 PC reaching out to my Security Onion's IP over port 8090 and is being met with a SYN_WAIT packet state so my endpoint is reaching out but is not being met with a connection response.

[Link to the video with timestamp](https://youtu.be/SAubxi13K-Q?t=2551)

**Comment by defensivedepth** [comment]
A few questions to help us help you:

1) What type of install is this? (EVA / Standalone / etc)

2) Do you see any errors when running `sudo so-status`

3) Do you see any failures / errors when running `sudo salt-call state.highstate`

**Reply by strf0x1** [reply]
Don't know if it helps, but I had to reboot my Win 10 hosts to see it in Fleet. Tested 3 or 4 times and it was the same each time.

**Reply by ghost** [reply]
>A few questions to help us help you:
>What type of install is this? (EVA / Standalone / etc)

Standalone

>Do you see any errors when running sudo so-status

Cortex was errored out but has been restarted without issue and aptcacherng is "missing".

>Do you see any failures / errors when running sudo salt-call state.highstate

[WARNING ] State for file: /etc/pki/ca.crt - Neither 'source' nor 'contents' nor 'contents_pillar' nor 'contents_grains' was defined, yet 'replace' was set to 'True'. As there is no source to replace the file with, 'replace' has been set to 'False' to avoid reading the file unnecessarily.
[WARNING ] State for file: /etc/pki/ca.crt - Neither 'source' nor 'contents' nor 'contents_pillar' nor 'contents_grains' was defined, yet 'replace' was set to 'True'. As there is no source to replace the file with, 'replace' has been set to 'False' to avoid reading the file unnecessarily.

Followed by this output at the end

Summary for local
Succeeded: 490 (changed=14)
Failed: 0
Total states run: 490
Total run time: 256.753 s

**Reply by innovate-support** [reply]
I am having the same issues.  Server is v2.3.2  I don't see any hosts at all in Fleet, not even the SO Server.  I've run so-allow multiple times to allow not only the specific IP of the Win10 machine, but also the whole network range.  I can nmap from Win10 and see that port 8090 is open on the SO Server, but no hosts ever show up in Fleet even after reinstalling the msi-launcher.msi and rebooting both the Win10 machine and the SO Server.  Both are VM's that are on the same virtual network and are talking fine with each other over ports 80, 443, and 22.  All other tools seem to be working fine with Alerts, but no **osquery** logs are coming in.

1. I am running the server in Standalone mode.
2. sudo so-status shows all OK
3. sudo salt-call state.highstate shows the same 2 [WARNING] lines mentioned above.

**Reply by defensivedepth** [reply]
@innovate-support Can you please start a new Discussion?

**Reply by innovate-support** [reply]
Will start a new discussion as requested, but this one hasn't been successfully answered yet and I'm having the exact same issues, so I assumed you'd rather keep it here instead of me duplicating.
